### DISCUSSION TITLE
The Spy: a different play style for the main plot line

### DISCUSSION THREAD

**Discussion by dahe-at-google** [endless-sky/endless-sky#7878] [Ideas]
I'd like to propose a different way of navigating the main storyline by using espionage. It ends up at the same place as the main storyline with FirstLast being a well-known and respected hero by a different route, with less reliance on a military style of play.

This is a little half-baked, but let's see where it goes.

The idea in The Spy is to make espionage a key element in winning the campaign. Instead of going into the big battles with a fleet, you lay the groundwork for subverting them, working behind the scenes so that your opponent's fleet is less effective. Some possibilities for what this might look like:

- you spread misinformation so that the fleet heading to a battle gets split up (part of it gets ordered to the wrong place). The whole fleet arrives eventually, but split into pieces which can more easily be picked off by the free worlds.
- similarly, by planting information, you lead Parliament to direct more the the fleet to protect the Paradise worlds at a critical point in the campaign.
- a virus get placed into the Republic or Syndicate's onboard computers. When the battle starts and for a moment all the shield on the ships drop, before regenerating in the normal way. So there is a window to attack while they are vulnerable. This might be too much like the virus in the Wanderer/Kor campaign, so another option is a drone which you deploy from your ship and which sends the signal to drop the shields.
- sabotage the Syndicate nukes (like changing the command codes) so that instead of drawing fire as in the standard FW scenario, you transmit a code to disable them so that the ship with the nukes can be attacked and destroyed.

There may be subplots that can be adapted as well. For example, instead of capturing the ship with the electron beam, you do a daring covert operation on a Republic planet.

I think much of this can be implemented as a plugin, that is without any changes to the game engine, though things like the shield-cancelling drone might not be possible.

These proposals don't get rid of combat and the need for raw firepower entirely, but they make it less about who has the biggest blaster. I imagine that you would play the campaign with a small, fast ship like a scout and stay on the fringe of the big battles. Perhaps this could be enforced by only allowing the shield-cancelling drone or the nuke command code devices to be installed on
specialized models of certain ships, available only as the result of a side quest. 

Some possible game mechanics:

- missions which allow you to build up a network on various planets. Think of this as like a more fine grained version of reputation. For example, you have a network value of X from planet Y to planet Z, obtained by missions between the planets. When it come to trigger the misinformation campaign mentioned above, you want the misinformation to be spread on a Republic planet, but at the that point in the game you probably can't reach the Republic, so you trigger it on a planet you do have access to and which has a network to theRepublic planet.

- use the network or bribery to get the parts from Republic and Pirate planets, in order the assemble the shield-disabling drone or other gadgets use in the plot.

- espionage on behalf of pirate worlds (like getting the information about convoys they might raid) so that for one or more of the big battles they come in as temporary allies.

Other ideas:

- another thread of early campaign missions instead of the Escort/Recon/Bounty missions based on intelligence gathering. Also useful as a way of introducing networks.

- instead of knowing where Alexander Soylent it, you have to track him down. This could happen during the current FW Reconciliation Break.

I don't have a lot of experience of writing plugins (and possibly not the time to do so), so I am throwing this idea out for reactions and comments and in case anyone more talented than I am wants to take it up. I may have a go at it if I can.

**Comment by Zitchas** [comment]
I don't know  how feasible it would be to implement, but it does sound like a fun alternative to the military-heavy plot line.

**Comment by warp-core** [comment]
@Zitchas are you able to do something more about this spambot thing?

**Reply by Zitchas** [reply]
Well, somewhat. I can (and have) blocked the spamming accounts from the organization, and that automatically hides all their posts as "spam."
